modules/llm-cache/tests/kv_state_cache_test.cc

```diff
@@ -44,8 +44,13 @@ KVStateCacheManager* kv_state_cache_manager;
 
 void init(int dimension, int capacity, int layer, int block_size,
           std::string socket) {
-  kv_state_cache_manager = new KVStateCacheManager(dimension, capacity, layer,
-                                                   block_size, 3, socket);
+  Status status = KVStateCacheManager::MakeKVStateCacheManager(
+      kv_state_cache_manager, dimension, capacity, layer, block_size, 3,
+      socket);
+  if (!status.ok()) {
+    LOG(ERROR) << "Failed to create KVStateCacheManager: " << status.ToString();
+    exit(1);
+  }
```

Comment:
Use `VINEYARD_CHECK_OK(KVStateCacheManager::Make(...))` in test cases.

Reply:
OK